pkg/aflow/tool/toolkit: broke non-linux builds

OpenBSD instances are failing with:
```
--- FAIL: TestToolkitsInTestData (2.13s)
    --- FAIL: TestToolkitsInTestData/toolkit_functional_test.c (1.11s)
        toolkit_test.go:33: compilation failed: exit status 1
            In file included from testdata/toolkit_functional_test.c:2:
            testdata/../race_toolkit.h:14:10: fatal error: 'linux/futex.h' file not found
               14 | #include <linux/futex.h>
                  |          ^~~~~~~~~~~~~~~
            1 error generated.
    --- FAIL: TestToolkitsInTestData/toolkit_standalone_test.c (1.01s)
        toolkit_test.go:33: compilation failed: exit status 1
            In file included from testdata/toolkit_standalone_test.c:4:
            testdata/../race_toolkit.h:14:10: fatal error: 'linux/futex.h' file not found
               14 | #include <linux/futex.h>
                  |          ^~~~~~~~~~~~~~~
            1 error generated.
```

cc @blackgnezdo

--- a/pkg/aflow/tool/toolkit/toolkit_test.go
+++ b/pkg/aflow/tool/toolkit/toolkit_test.go
@@ -1,6 +1,8 @@
 // Copyright 2026 syzkaller project authors. All rights reserved.
 // Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
 
+//go:build linux
+
 package toolkit
 
 import (

